html-css/kerry/navigation-flex.css

```
@@ -0,0 +1,56 @@
+body {
+	background: #F4F4F4 ;
+}
+#big {
+	display: flex;
+	flex-direction: column;
+	width: 1300px;
+	height: 265px;
+	margin: 160px auto;
```

```suggestion
	margin-top: 160px;
	margin-left: auto;
	margin-right: auto;
```

因为呢，`margin-bottom` 没必要设置咧~